src/block_metadata.rs

```diff
@@ -33,11 +36,22 @@ impl<I: NodeType> Committable for HashableBlock<I> {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq, Hash)]
 #[serde(bound = "")]
 pub struct BlockMetadata<I: NodeType> {
```

Comment:
Unrelated to this change, but I'm thinking we should probably rename this to disambiguate with respect to `BlockPayload::Metadata`.

Maybe `AvailableBlockInfo` would work...

Reply:
Done!